src/gibbs.c

```diff
@@ -666,15 +669,22 @@ SEXP collapsedGibbsSampler(SEXP documents,
 
     for (ii = 0; ii < length(initial_); ++ii) {
       if (!strcmp(CHAR(STRING_ELT(names, ii)), "assignments")) {
-        initial = VECTOR_ELT(initial_, ii);
-        CHECKLEN(initial, NewList, nd);
+    	  // srecko commented this
+    	  //initial = VECTOR_ELT(initial_, ii);
+
+    	  //CHECKLEN(initial, NewList, nd);
+
       } else if (!strcmp(CHAR(STRING_ELT(names, ii)), "topic_sums")) {
         initial_topic_sums = VECTOR_ELT(initial_, ii);
         if (!isInteger(initial_topic_sums) ||
             INTEGER(GET_DIM(initial_topic_sums))[0] != K ||
             INTEGER(GET_DIM(initial_topic_sums))[1] != length(V_)) {
           error("Initial topic sums must be a K x length(V) integer matrix.");
         }
+      } else if (!strcmp(CHAR(STRING_ELT(names, ii)), "net.assignments.left")) {
+    	  initial_net_left = VECTOR_ELT(initial_, ii);
```

Comment:
Just a nit but it would be nice if everything were indented with increments of 2 spaces.

Reply:
Honest, I thought it was... probably messed up something.